vscode-dotnet-runtime-library/src/Acquisition/InstallScriptAcquisitionWorker.ts

```diff
@@ -18,7 +18,7 @@ import { IInstallScriptAcquisitionWorker } from './IInstallScriptAcquisitionWork
 
 export class InstallScriptAcquisitionWorker implements IInstallScriptAcquisitionWorker {
     protected webWorker: WebRequestWorker;
-    private readonly scriptAcquisitionUrl: string = 'https://aka.ms/dotnet-install-script-';
+    private readonly scriptAcquisitionUrl: string = 'https://dot.net/v1/dotnet-install.';

```

Comment:
Hmmm, this is an interesting choice. I get the harm in redirects; however, having an aka.ms link allows us to move that link whenever we want. Do we have a guarantee that the `https://dot.net/v1/dotnet-install` is stable?

Reply:
Yes, when I originally added this redirect there wasn't much infrastructure around the install scripts but now they have a multi-teired system of releasing new changes, so that this link is only updated once the new script has been dogfooded by the other tiers first.